client/src/assets/components/profile/EditProfile.jsx

```diff
@@ -700,9 +739,37 @@ const EditProfile = () => {
         )}
       </div>
       {/* End Skills Field */}
-
+<<<<<<< HEAD
+      {/* Toast components */}
+      <ErrorToast
+        message={errorMessage}
+        show={showErrorToast}
+      />
+      <SuccessToast
+        message={successMessage}
+        show={showSuccessToast}
+=======
```

Comment:
Not sure if this merge issue popped up after you committed, but if it's just GitHub I can fix it. Just wanted your feedback on it.